src/coreclr/vm/methodtablebuilder.cpp

```diff
@@ -6599,7 +6599,7 @@ VOID MethodTableBuilder::PlaceInterfaceDeclarationOnClass(
     DispatchMapTypeID firstDispatchMapTypeID;
     ComputeDispatchMapTypeIDs(
         pDeclMT,
-        &pDecl->GetMethodSignature().GetSubstitution(),
+        pDecl->GetMethodSignature().GetSubstitution(),
```

Comment:
Looks like there are a few other instances of this function call under `#if DEBUG`, which needs this fix `git grep -n 'GetSubstitution('`

Reply:
Boo. I thought I was building under debug locally. :(